core/src/main/java/com/sequenceiq/cloudbreak/core/bootstrap/service/ClusterBootstrapper.java

```diff
@@ -361,4 +373,21 @@ private void validatePollingResultForCancellation(PollingResult pollingResult, S
         }
     }
 
+    String getNodeNameForInstance(InstanceMetaData im, Stack stack,
```

Comment:
this should be private or moved to a separate class

Reply:
ok